dds_web/scheduled_tasks.py

```diff
@@ -1,24 +1,74 @@
 import flask_apscheduler
 import flask
 
-# Apscheduler
+## Apscheduler
 scheduler = flask_apscheduler.APScheduler()
 
 
+# @scheduler.task("cron", id="available_to_expired", minute=5, hour=2, misfire_grace_time=1)
```

Comment:
I think we should plan the cron jobs for just after midnight. Multiple cronjobs should be running concurrently. We may need to tweak the queries a bit to block multiple queries happening at the same time for the same row.
```suggestion
# @scheduler.task("cron", id="available_to_expired", hour=0, minute=1, misfire_grace_time=3600)
```

Reply:
👍